### PR TITLE
Initial module creation

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: security_hub_accepter.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+
+def accept_pending_invite(sechub, master_account_id, invite_id):
+    sechub.accept_invitation(MasterId=master_account_id, InvitationId=invite_id)
+
+
+def main(
+    master_account_id, remove_master=False, profile=None, role_arn=None, region=None
+):
+    region = region or None
```
Comment (userhas404d) [inline security_hub_accepter.py:104]: isn't this redundant, or am I missing something?

Comment (confusdcodr) [reply security_hub_accepter.py]: Yeah, idk. Some copy pasta magic. Removed.

Comment (confusdcodr) [reply security_hub_accepter.py:104]: So tried it without it and it fails so not sure why this is necessary. @lorengordon any insights?

Comment (lorengordon) [reply security_hub_accepter.py:104]: terraform passes the values through as empty strings. . . . this coerces them back to `None`

**Pull request merged**